kubemarine/kubernetes/__init__.py

```diff
@@ -97,6 +97,35 @@ def generic_upgrade_inventory(cluster: KubernetesCluster, inventory: dict) -> di
     return inventory
 
 
+def enrich_restore_inventory(inventory: dict, cluster: KubernetesCluster) -> dict:
+    if cluster.context.get("initial_procedure") != "restore":
+        return inventory
+
+    logger = cluster.log
+    kubernetes_descriptor = cluster.context['backup_descriptor'].setdefault('kubernetes', {})
+    initial_kubernetes_version = get_initial_kubernetes_version(inventory)
+    backup_kubernetes_version = kubernetes_descriptor.get('version')
+    if not backup_kubernetes_version:
+        logger.debug("Not possible to verify Kubernetes version, as descriptor does not contain 'kubernetes.version'")
```

Comment:
Maybe it should be warning?

Reply:
This logging was moved from restore.py as-is, but I agree to change level. Changed.